Updated MixedReality250 project, matched to MRTK Dev_Unity_2017.2.0

SimonDarksideJ opened the pull request:
This needs to be pushed to a new Dev branch, whilst the Dev MRTK is being worked on.

*Note
Outstanding tasks
* Finish abstracting SharingWithUNET\PlayerController.cs
* Needs new fixes to MRTK SharingWithUNET\UNetAnchorManager (In this source, to be submitted to MRTK)

SimonDarksideJ committed:
Updated MixedReality250 project, matched to MRTK Dev_Unity_2017.2.0

*Note
Outstanding tasks
* Finish abstracting SharingWithUNET\PlayerController.cs
* Needs new fixes to MRTK SharingWithUNET\UNetAnchorManager (In this source, to be submitted to MRTK)

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/MixedReality250?pullRequest=10) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/MixedReality250?pullRequest=10) before we can accept your contribution.<br/><br/>:x: DDReaper [sign now](https://cla.opensource.microsoft.com/Microsoft/MixedReality250?pullRequest=10)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/MixedReality250?pullRequest=10) it.</sub>

SimonDarksideJ committed:
Updated MRTK from the weeks changes (also forgot to add a lot of the scripts in the last commit :S)
All Chapter scenes test ok with the exception of Ch2, which complains about the WorldAnchorManager, fine in others.

SimonDarksideJ committed:
Added AudioManager prefab to enable more reuse

SimonDarksideJ committed:
Minor tweaks. Also disabled the LevelSettings requirement for the puzzles to demonstrate interactions.  Ideally this just needs updating later to work with/without the sharing functionality.

Also had to reset the ProjectSettings as they freaked Unity out somehow.

darax commented:
wow!  I appreciate the enthusiasm!  To be honest, I wasn't expecting anyone to be trying to contribute at this point-for the other academy courses we haven't been accepting pull requests.  Just recently the tool chain has become good enough for me to work on this again, and I'd like to move quickly without accidentally stepping on someone else's hard work.

darax commented:
also, if there are things from this that you really need me to get in (like are the project settings still making unity mad?) I'll do it in my next push.

DDReaper commented:
HI @darax No worries.
I had to do this as I'm using the content for a new "Mixed Reality" workshop for immersive devices this week.  As I already did the work, I simply reciprocated all the teams efforts by pushing back.

If you want to wait and improve, by all means, use or not use :D

DDReaper commented:
P.S. There are still some big changes going on with the dev / immersive branch at the moment, so it will need another update later.
Hence why I recommend this goes in to it's own dev Branch rather than merge in to master.
